src/jobs/release-package.yml

```diff
@@ -13,6 +13,10 @@ description: >
     - SALESFORCE_KEY
   The following environment variables are required if creating a github release:
     - GH_TOKEN
+  The following environment variables are required for CTC
+    - SF_CHANGE_CASE_SFDX_AUTH_URL
+    - SF_CHANGE_CASE_TEMPLATE_ID
+    - SF_CHANGE_CASE_SCHEDULE_BUILD
```

Comment:
Is the implementation step required or optional?

Reply:
I think it's required by the GUS side.  The plugin retrieves them because they're child records of the template case.  I **think** the plugin will fuss if there aren't any.  If it doesn't, GUS will.

Reply:
I need to add that the Configuration_Item is now required